checkpoint/provider.go

```diff
@@ -465,7 +475,7 @@ func providerConfigure(data *schema.ResourceData) (interface{}, error) {
 	}
 }
 
-func login(client *checkpoint.ApiClient, username string, pwd string, apiKey string, domain string, sessionName string, sessionDescription string) (Session, error) {
+func login(client *checkpoint.ApiClient, username, pwd, apiKey, domain, sessionName, sessionDescription string, sessionTimeout int) (Session, error) {
```

Comment:
It is more readable when there is type next to each parameter. 
Change it back to the old format please.